config/management/operational/src/validator_config.rs

```diff
@@ -37,10 +37,11 @@ impl SetValidatorConfig {
             self.validator_config.validator_backend.name(),
             &self.validator_config.validator_backend.validator_backend,
         )?;
+        let operator_account = storage.account_address(OPERATOR_ACCOUNT)?;
         let owner_account = storage.account_address(OWNER_ACCOUNT)?;
 
         let client = JsonRpcClientWrapper::new(self.host);
-        let sequence_number = client.sequence_number(owner_account)?;
+        let sequence_number = client.sequence_number(operator_account)?;
```

Comment:
Curious, what's the difference between these sequence numbers?

Reply:
From what I understand, sequence numbers are monotonically increasing numbers associated with accounts. Each transaction submitted and signed by an account must include this sequence number, to prevent against replay attacks. Given that the set_validator_config transaction is submitted by the operator account, we should read the sequence number of the operator_account (the owner account is a separate account 😄)